unit-testing/presentation.html

```diff
@@ -2,43 +2,512 @@
-    <link rel="stylesheet" href="../reveal-js/common.css" />
+    <link
+      rel="stylesheet"
+      href="https://rolling-scopes-school.github.io/react-native-course/reveal-js/plugin/highlight/monokai.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://rolling-scopes-school.github.io/react-native-course/reveal-js/common.css"
+    />
+    <style class="jsStyle"></style>
   </head>
   <body>
     <div class="reveal">
-      <!-- Any section element inside of this container is displayed as a slide -->
       <div class="slides">
-        <section class="center">
-          <h2 class="custom-presentation-name"> Presentation​ Name </h3>
+        <!-- 0 slide -->
+        <section class="presentation-name-slide">
+          <h4>Unit tests</h4>
+        </section>
+        <!-- 1 slide -->
+        <section class="agenda-slide">
+          <h4 class="slide-name">AGENDA</h4>
+          <ul>
+            <li class="fragment">Testing in software development</li>
+            <li class="fragment">Testing Pyramid</li>
+            <li class="fragment">Unit tests</li>
+            <li class="fragment">TDD & BDD</li>
+            <li class="fragment">Testing statistic</li>
+            <li class="fragment">Testing libraries</li>
+            <li class="fragment">Jest as a testing framework</li>
+            <li class="fragment">Husky</li>
+          </ul>
+        </section>
+        <!-- 2 slide -->
+        <section class="bullets-slide">
+          <h4 class="slide-name">Testing role in Software Development​</h4>
+          <ul>
+            <li class="fragment">
+              Testing is about verifying that what was specified is what was
+              delivered.
+            </li>
+            <li class="fragment">
+              The testing is important since it discovers defects/bugs before
+              the delivery to the client, which guarantees the quality of the
+              software.​
+            </li>
+          </ul>
+        </section>
+        <!-- 3 slide -->
+        <section class="bullets-slide">
+          <h4 class="slide-name">Major benefits of software testing​</h4>
+          <ul>
+            <li class="fragment">Launch a flawless software application</li>
+            <li class="fragment">Prevent targeted malware attacks</li>
+            <li class="fragment">Improves the user experience</li>
+            <li class="fragment">Curtail maintain costs</li>
+            <li class="fragment">Improve time-to-market</li>
+            <li class="fragment">Enhance quality of apllication</li>
+            <li class="fragment">Boost competitiveness</li>
+          </ul>
+        </section>
+        <!-- 4 slide -->
+        <section class="bullets-slide">
+          <h4 class="slide-name">The scope of Testing​</h4>
+          <ul>
+            <li class="fragment">
+              The test scope defines what areas, features, and functionalities
+              of an application should be tested
```

Comment:
This text is too long and doesn't really shows anything. It must be in notes. This slide need pictures and a list with items.